specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```diff
@@ -0,0 +1,4472 @@
+          }
+        }
+      },
+      "required": [
+        "deviceId",
+        "state",
+        "healthChecks"
+      ]
+    },
+    "DeviceHealthState": {
+      "description": "Device health states",
+      "type": "string",
+      "enum": [
+        "healthy",
+        "unhealthy"
+      ],
+      "x-ms-enum": {
+        "name": "DeviceHealthState",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "healthy",
+            "name": "healthy",
+            "description": "Agent is healthy"
+          },
+          {
+            "value": "unhealthy",
+            "name": "unhealthy",
+            "description": "Agent is in an unhealthy state"
+          }
+        ]
+      }
+    },
+    "DeviceHealthList": {
+      "description": "Array of Device Health, with server paging support.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of pageable items.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeviceHealth"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of items.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "HealthCheck": {
+      "description": "Health check",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Health check name"
+        },
+        "result": {
+          "$ref": "#/definitions/HealthCheckResult",
+          "description": "Health check result"
+        }
+      }
+    },
+    "DeviceHealthFilter": {
+      "type": "object",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/DeviceHealthState",
+          "description": "Device health state"
+        },
+        "deviceId": {
+          "type": "string",
+          "description": "Device Id"
+        },
+        "moduleId": {
+          "type": "string",
+          "description": "Module Id"
+        }
+      },
+      "description": "Device health filter."
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of the API to be used with the client request."
+    },
+    "EndpointParameter": {
+      "name": "endpoint",
+      "in": "path",
+      "required": true,
+      "format": "url",
+      "type": "string",
+      "x-ms-skip-url-encoding": true,
+      "description": "Account endpoint.",
+      "x-ms-parameter-location": "client",
+      "x-ms-parameter-grouping": {
+        "name": "account-options"
+      }
+    },
+    "InstanceIdParameter": {
+      "name": "instanceId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-skip-url-encoding": true,
+      "description": "Account instance identifier.",
```

Comment:
We should aim to be more specific here - what type of account, and what is meant by instance identifier (the Azure resource ID? IoT Hub name? some other service-specific ID?)

It causes user confusion, e.g. https://stackoverflow.com/questions/71673552/where-can-i-find-iot-hub-device-update-instanceid